cmd/cli/process.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+/*
+process a scorecard document
+*/
+import (
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/NOAA-GSL/vxDataProcessor/pkg/manager"
+	"github.com/joho/godotenv"
+)
+
+func main() {
+	os.Exit(process())
+}
+
+func process() int {
+	defer fmt.Println("Finished")
+	if len(os.Args) != 2 {
+		fmt.Println("Usage:", os.Args[0], "document_id")
+		return 1
+	}
+
+	documentId := os.Args[1]
+	start := time.Now()
+	// load the ${HOME}/.env if it exists
+	environmentFile := fmt.Sprint(os.Getenv("HOME"), "/.env")
+	err := godotenv.Load(environmentFile)
```

Comment:
The "go dot env" library will automatically look for `./.env`. I'd be in favor of simplifying this to just:

```suggestion
	err := godotenv.Load()
```

We'd have the `.env` file in the project directory while we were developing but it's been added to the `.gitignore` file so we'd have to explicitly `git add .env` it to get it into the repo.

Reply:
I did that so that we can override the .env for special circumstances like debugging. For example if I run the MATS scorecard in my debugger the url changes to localhost:3000 (no appname) so it made sense to me to allow to have mutliple env files that provide different urls for different purposes.

Reply:
I think in that case, we could just change the `.env` file contents for debugging locally. We could also have multiple `.env` files (e.g. `.env.local`, `.env.somespecialsituation`) that we either copy in and out of `.env` or symlink to `.env`.

If you did want an environment variable to control which environment file to load, we could do something like the following which would test for a particular ENV variable. E.g. you'd set something like `PROC_ENV_PATH=/path/to/some/env_file`. 

```go
environmentFile, set := os.Lookupenv("PROC_ENV_PATH")
if !set {
	err := godotenv.Load() // Loads from "$(pwd)/.env"
} else {
	err := godotenv.Load(environmentFile) // Loads from whatever PROC_ENV_PATH has been set to
}
```

The tricky part with the current way this is written is that it's hardcoded to the user's HOME directory which isn't normally where `.env` files go as you'd only be able to have one per system. Usually, `.env` files are placed in the root of the git repo and added to the `.gitignore` file so they aren't checked in. Each developer can then modify the `.env` file as needed for their local environment.